backend/src/db/migrate/store.js

```diff
@@ -1,11 +1,62 @@
 import { getDriver, getNeode } from '../../db/neo4j'
+import { hashSync } from 'bcryptjs'
+import { v4 as uuid } from 'uuid'
+
+const defaultAdmin = {
+  email: 'admin@example.org',
+  password: hashSync('1234', 10),
+  name: 'admin',
+  id: uuid(),
+  slug: 'admin',
+}
+
+const createDefaultAdminUser = async (session) => {
+  const readTxResultPromise = session.readTransaction(async (txc) => {
+    const result = await txc.run(
+      'MATCH (email:EmailAddress)-[:BELONGS_TO]->(user:User) RETURN count(user) AS userCount',
```

Comment:
Why do you use the EMail relation here?
is it not sufficient to just count the users?